packages/xrpl/test/wallet/batchSigner.test.ts

```diff
@@ -0,0 +1,370 @@
+import { assert } from 'chai'
+
+import {
+  Batch,
+  decode,
+  ECDSA,
+  encode,
+  ValidationError,
+  Wallet,
+} from '../../src'
+import {
+  BatchFlags,
+  BatchInnerTransaction,
+  BatchSigner,
+} from '../../src/models/transactions/batch'
+import {
+  combineBatchSigners,
+  signMultiBatch,
+} from '../../src/Wallet/batchSigner'
+
+const secpWallet = Wallet.fromSeed('spkcsko6Ag3RbCSVXV2FJ8Pd4Zac1', {
```

Comment:
Can you use these variables when you construct transaction payloads? Visually, I can't understand the goal of these tests.

Reply:
What do you mean?